recipes/pygraphblas/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pygraphblas" %}
+{% set version = "3.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/michelp/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: db0e8a6e0370c9447ea128ca26e27f5df4a9d711c4c3037c661f0ef451ba01ea
+
+build:
+  skip: true  # [win or py<37]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - setuptools
+    - cffi
+    - graphblas
+    - pytest-runner
+  run:
+    - python
+    - cffi
+    - numba
+
+test:
+  imports:
+    - pygraphblas
+  commands:
+    - python -c "from pygraphblas import Vector, INT64 ; v = Vector.sparse(INT64, 10) ; assert v.size == 10"
+
+about:
+  home: https://github.com/michelp/pygraphblas
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'GraphBLAS for Python'
+  description: |
+    pygraphblas is a Python wrapper around the GraphBLAS API.
+  dev_url: https://github.com/michelp/pygraphblas
+
+extra:
+  recipe-maintainers:
+    - eriknw
+    - jim22k
```

Comment:
CC @jim22k - are you willing to be a maintainer?

Reply:
Yes